suites/pacific/cephadm/test-container-cli-args.yaml

```diff
@@ -0,0 +1,373 @@
+                        service_id: my-rgw
+                        placement:
+                          count_per_host: 2
+                          nodes:
+                            - node4
+                            - node3
+                        spec:
+                          rgw_frontend_port: 8080
+                          rgw_realm: east
+                          rgw_zone: india
+                        extra_container_args:
+                          - "--cpus=1"
+                - config:
+                    command: shell
+                    args: # sleep to get all services deployed
+                      - sleep
+                      - "120"
+      desc: Deploy MDS, NFS and RGW services parallely
+  - test:
+      name: Setup destination node for SNMP traps
+      desc: Install snmptrapd tool and install CEPH MIB on destination node
+      module: snmp_destination.py
+      polarion-id: CEPH-83574775
+      config:
+        node: node3
+        specs:
+              - service_type: snmp-destination
+                spec:
+                  credentials:
+                    snmp_v3_auth_username: myadmin
+                    snmp_v3_auth_password: mypassword
+  - test:
+      name: SNMP Gateway Service deployment with spec
+      desc: Add SNMP Gateway services using spec file
+      module: test_cephadm.py
+      polarion-id: CEPH-83574887
+      config:
+        steps:
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: snmp-gateway
+                  service_name: snmp-gateway
+                  placement:
+                    count: 1
+                  spec:
+                    credentials:
+                      snmp_v3_auth_username: myadmin
+                      snmp_v3_auth_password: mypassword
+                    port: 9464
+                    snmp_destination: node3
+                    snmp_version: V3
+                  extra_container_args:
+                    - "--cpus=1"
+          - config:
+              command: shell
+              args: # sleep to get all services deployed
+                - sleep
+                - "120"
+  - test:
+      name: Configure client
+      desc: Configure client on node4
+      module: test_client.py
+      polarion-id:
+      config:
+        command: add
+        id: client.1                      # client Id (<type>.<Id>)
+        node: node4                       # client node
+        install_packages:
+          - ceph-common                   # install ceph common packages
+        copy_admin_keyring: true          # Copy admin keyring to node
+        store-keyring: true               # /etc/ceph/ceph.client.1.keyring
+        caps:                             # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
```

Comment:
can we remove this test case, if we are covering in another test suite already.

Reply:
Done.